app/code/Magento/Catalog/Helper/Product/ProductList.php

```diff
@@ -29,6 +29,7 @@ class ProductList
      * @var \Magento\Framework\App\Config\ScopeConfigInterface
      */
     protected $scopeConfig;
+    protected $coreRegistry;
```

Comment:
Firstly there is a type hint missing here. Secondly does this need to be protected? My suggestion would be that this can be made private to start with as it is only being used in this class.